src/integration/java/uk/nhs/prm/repo/ehrtransferservice/utils/SQSQueueUtility.java

```diff
@@ -8,12 +8,12 @@
 import org.springframework.stereotype.Component;
 
 @Component
-public final class QueueUtility {
+public final class SQSQueueUtility {
```

Comment:
Could we see us ever adding any ActiveMQ queue utility methods? Happy to keep this as `SQSQueueUtility`, just looking into the future 🧙

Reply:
I did this because I went poking around on this class looking for utility methods that were for the ActiveMQ. I discovered pretty quickly that the only utility methods here related to the SQS queues and wanted to rename it to avoid the same mistake. I'm not overly precious but I figured that this class will only ever be for SQS queue utility methods.

Reply:
Fair enough! Happy with this :)